library/src/main/java/com/cottacush/android/currencyedittext/CurrencyEditText.kt

```diff
@@ -108,4 +108,16 @@ class CurrencyEditText(context: Context, attrs: AttributeSet?) : TextInputEditTe
             if (text.toString() == currencySymbolPrefix) setText("")
         }
     }
+
+
```

Comment:
It's null probably because setText/SetSelection/onSelectionChanged/etc is being called during initialization of the super classes, in the chain of calls to the super constructors.

Reply:
Yes, probably. Do you want me to remove comment, or should we keep it?

Reply:
You can remove it.